server/game/cards/19.1-AS01/Crane/DevelopingMasterpiece.js

```diff
@@ -33,15 +29,26 @@ class DevelopingMasterpiece extends DrawCard {
         this.action({
             title: 'Gain honor',
             phase: Phases.Fate,
-            condition: context => context.source.parent,
-            cost: [
-                captureParentCost(),
-                AbilityDsl.costs.removeSelfFromGame()
-            ],
-            gameAction: AbilityDsl.actions.gainHonor(context => ({
-                amount: context.costs.captureParentCost ? context.costs.captureParentCost.getGlory() : context.source.parent.getGlory(),
+            condition: (context) => context.source.parent,
+            cost: [captureParentCost(), AbilityDsl.costs.removeSelfFromGame()],
+            gameAction: AbilityDsl.actions.gainHonor((context) => ({
+                amount: this._masterpieceHonorAmount(context),
                 target: context.player
-            }))
+            })),
+            effect: 'gain {1} honor',
+            effectArgs: (context) => [this._masterpieceHonorAmount(context)],
+            handler: (context) => {
```

Comment:
Probably better to put the handler in a `then:` block, would stop this from being able to fire and doing nothing if you can't gain any honor.